src/main/scala/nl.knaw.dans.easy.umd/Transformer.scala

```diff
@@ -49,13 +50,35 @@ object Transformer {
             Success(())
         }
       case ("EMD", "orgISNI") =>
-        if ((oldXML \\ "organization").exists(_.text == expectedOldValue))  Success(())
+        if ((oldXML \\ "organization").exists(_.text == expectedOldValue)) Success(())
         else Failure(new NotImplementedError(s"no organization with name [$expectedOldValue] found."))
       case _ =>
         Success(())
     }
   }
 
+  private def addChildTransformer(label: String, newChild: Node): RuleTransformer =
+    new RuleTransformer(new RewriteRule {
+      override def transform(n: Node): Seq[Node] = n match {
+        case n @ Elem(_, `label`, _, _, _*) => addChild(n, newChild)
+        case other => other
+      }
+    })
+
+  private def addChild(n: Node, newChild: Node) = n match {
+    case Elem(prefix, label, attribs, scope, child @ _*) if !childExists(child, newChild) =>
+      Elem(prefix, label, attribs, scope, false, child ++ newChild: _*)
+    case other => other
+  }
```

Comment:
You can merge these two methods. The first case in the `RewriteRule` is about the same element as the first case in `addChild`.

```suggestion
  private def addChildTransformer(label: String, newChild: Node): RuleTransformer =
    new RuleTransformer(new RewriteRule {
      override def transform(n: Node): Seq[Node] = n match {
        case Elem(prefix, `label`, attribs, scope, child @ _*) if !childExists(child, newChild) => Elem(prefix, label, attribs, scope, false, child ++ newChild: _*)
        case other => other
      }
    })
```